src/templates/admin/copyediting/copyediting.html

```diff
@@ -25,21 +26,21 @@ <h2>Articles in Copyediting</h2>
                     <th>Section</th>
                     <th>Projected Issue</th>
                     <th>Stage</th>
-                    <th></th>
+                    <th>Assignments</th>
                 </tr>
                 </thead>
                 <tbody>
                 {% for article in articles_in_copyediting %}
                     <tr>
                         <td>{{ article.pk }}</td>
-                        <td>{{ article.safe_title }}</td>
+                        <td><a href="{% url 'article_copyediting' article.pk %}">{{ article.safe_title }}</a></td>
                         <td>{{ article.date_submitted }}</td>
                         <td>{{ article.date_accepted }}</td>
                         <td>{{ article.correspondence_author.full_name }}</td>
                         <td>{{ article.section.name }}</td>
                         <td>{% if article.projected_issue %}{{ article.projected_issue.display_title }}{% else %}None{% endif %}</td>
                         <td>{{ article.stage }}</td>
-                        <td><a href="{% url 'article_copyediting' article.pk %}">View</a></td>
+                        <td>{% for assignment in article.copyeditassignment_set.all %}<a href="{% url 'editor_review' article.pk assignment.pk %}">{{ assignment.copyeditor.full_name }}</a>{% if not forloop.last %}, {% endif %}{% endfor %}</td>
```

Comment:
This will leave a blank cell if there are no copyeditors assigned.

For tables to be accessible there should not be any empty cells.

Please provide a default entry for when there are no copyeditors assigned.  This does not have to be visible, it could be just for screen readers.

Reply:
Good catch, added an `{% empty %}` statement.

Reply:
Ah, good catch back at you - I think that is a different problem to the one I saw.

The `{% empty %}` statement looks like one for a row.  I was concerned about an empty cell.  For the case where an article had yet to have any copyeditors assigned.